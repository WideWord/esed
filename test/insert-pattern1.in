acd
awdpokw
wefpoakw
acd
addddfff
