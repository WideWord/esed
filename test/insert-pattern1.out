1234
acd
awdpokw
wefpoakw
1234
acd
addddfff
